subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/FileContentGenerator.groovy

```diff
@@ -59,6 +61,15 @@ abstract class FileContentGenerator {
         }
         ${dependenciesBlock('api', 'implementation', 'testImplementation', subProjectNumber, dependencyTree)}             
 
+        allprojects {
+            dependencies{
+        ${
+            language == Language.GROOVY ? directDependencyDeclaration('implementation', 'localGroovy()') : ""
```

Comment:
Maybe we should use a different `Groovy` version that `localGroovy`. I think nobody who does Groovy development uses `localGroovy`. WDYT?

Reply:
Well, I did. When I developed some little hobby project, I always use `localGroovy()`. I guess the real problem is, if we're comparing two Gradle versions, the potential Groovy version difference between these Gradle versions might interfere the performance test.

Reply:
Yeah, that is a good reason to fix the Groovy version for this performance test. Then we need to remember to update the Groovy version from time to time.